### ISSUE TITLE
Flapping RIPE NCC RPKI Validator 3 test result

### ISSUE THREAD

**Issue by ximon18** [NLnetLabs/rpki-deploy#12]
Sometimes the NCC test succeeds very quickly, other times even after 24 retries 10 seconds apart it still fails. An initial improvement could be to check some property of the REST API output of the NCC validator to see if it is even "ready" and to not fail the test if it was not ready, only if it was ready and the ROAs are not equal to those of Krill.

**Comment by ximon18** [comment]
A couple of candidate REST API output prorperties that could be checked which seem to transition from false to true when our E2E test passes from fail to succeed:

1. `http://<FQDN>:8080/api/healthcheck`
```
{
  "data" : {
    "overalStatus" : "OK",
    "trustAnchorReady" : {
      "ta.tal" : false  <-----------
    },
```

2. `http://nccapicheck.krill.cloud:8080/api/trust-anchors`
```
{
  "data" : {
    "type" : "trust-anchor",
    "id" : 1,
    "name" : "ta.tal",
    "locations" : [ "rsync://nccapicheck.krill.cloud/repo/ta/ta.cer" ],
    "subjectPublicKeyInfo" : "MIIB..AQAB",
    "preconfigured" : true,
    "initialCertificateTreeValidationRunCompleted" : true,   <-----------
```

**Comment by ximon18** [comment]
Fixed by 1396c5a9202625ba7914126b4f7c1359c1c00434.
